test/fixtures/07-releases/releases.json

```diff
@@ -6,7 +6,8 @@
 		"status": "success",
 		"release_version": "v1.0.0",
 		"composition": {},
-		"source": "cloud"
+		"source": "cloud",
```

Comment:
I think these added props are not used since
See: https://github.com/balena-io/open-balena-api/blob/eec61b31174680015f7b8ecb671e2641a21404ba/test/test-lib/fixtures.ts#L169-L179